src/components/common/dialogs/showSupportDialog.js

```diff
@@ -1,30 +1,35 @@
 //@flow
+
 import React from 'react'
 import Text from '../view/Text'
 
-export const showSupportDialog = (showErrorDialog, hideDialog, push, message) => {
-  showErrorDialog(
-    message ? message + '. Please try again.' : `Something went wrong on our side. Please try again later.`,
-    undefined,
-    {
-      boldMessage: (
-        <>
-          <Text fontWeight="inherit" color="inherit">
-            {'Or contact '}
-          </Text>
-          <Text
-            fontWeight="inherit"
-            textDecorationLine="underline"
-            color="inherit"
-            onPress={() => {
-              hideDialog()
-              push('Support')
-            }}
-          >
-            support
-          </Text>
-        </>
-      ),
-    },
-  )
+export const showSupportDialog = (
+  showErrorDialog,
+  hideDialog,
+  push,
+  message = 'Something went wrong on our side',
+  addTryAgainSuffix = true,
+) => {
+  const displayMessage = addTryAgainSuffix ? message + '. Please try again.' : message
```

Comment:
i think addTryAgainSuffix  isn't needed
 "Please try again." could be just a part of the message

Reply:
then we should duplicate 'Please try again' text in every place where suport dialog is used.

Reply:
@yaroslav-fedyshyn-nordwhale that's ok. and less complicated. so pls revert addTryAgainSuffix